README.rst

```diff
@@ -264,7 +264,7 @@ SUSE family
 
 - openSUSE Leap 42.2/42.3
 - openSUSE Tumbleweed 2015
-- SUSE Linux Enterprise Server 11 SP1/11 SP2/11 SP3/12
+- SUSE Linux Enterprise Server 11 SP4/12 SP2
```

Comment:
Actually, with the patch levels in SLES here, I've always thought the `/`s were a tad confusing. Would you mind changing this to `11 SP4, 12 SP2` or use an `and` instead of a comma?

Reply:
Yeah, sure.

Reply:
Done.